components/FullEventCard/FullEventCard.js

```diff
@@ -1,18 +1,29 @@
 import * as React from 'react';
 import { fetchDELETEevent } from '../../apiCalls';
+import { connect } from 'react-redux';
+import { updateTrigger } from '../../actions/actions';
+import { fetchPATCHevent } from '../../apiCalls';
 import { StyleSheet, View, TouchableOpacity, Text, Alert, Button } from 'react-native';
 
-export function FullEventCard({ navigation, route }) {
-  const { event } = route.params
+export function FullEventCard(props) {
+  const { event } = props.route.params
 
   const onCountMeInButtonPress = () => {
-    Alert.alert('Welcome to the Party! 🤙')
+    if (event.current_participant_count === event.max_participant_count) {
+      Alert.alert('Sorry, this event is full.')
```

Comment:
nice job with this error handling, this should make it easy to avoid sending unneccesary requests to the end point